src/sentry/static/sentry/app/views/performance/vitalDetail/index.tsx

```diff
@@ -108,8 +108,16 @@ class VitalDetail extends React.Component<Props, State> {
         ? undefined
         : (vitalNameQuery as WebVital);
 
+    // The vital details page will always have only one project selected
+    const project = projects.find(p => p.id === String(eventView.project));
+    const projectSlug = project?.slug;
```

Comment:
This is not true, the vital details page can be a cross project view.

Reply:
oh good catch...let me update it